recipes/common.rb

```diff
@@ -28,6 +28,8 @@
         file.each_line do |line|
           if line =~ /^TYPE="([^"]+)"/
             type = Regexp.last_match(1)
+            # wazuh centos packages incorrectly set type="manager" (WTF!?)
+            type = type == "manager" ? "server" : type
```

Comment:
Not yet cuz haven't tracked down the specfile to see if it's their doing or they're just using someone else's.  Will do so and add comment to code.  Agree this should get sorted out upstream because, to my knowledge, `manager` is not a legitimate choice.  Should just be `(local|server|agent)`.